website/blog/2020-08-31-63updates.md

```diff
@@ -0,0 +1,107 @@
+
+# Community Modules
+
+As with every big release, we're still working hard on getting important community modules supported on Windows.
+
+Come check out what we've enabled for you this release!
+
+## ReactNavigation
+
+![reactnavigation-rnwindows](assets/react-navigation-rnw.gif)
+
+The most well-used and much needed [react-navigation](https://github.com/react-navigation/react-navigation/tree/main/packages/native) now has full support on Windows. Navigate between screens with native ease and preserve your stack history in this amazing community module.
+
+## CheckBox
+
+![checkbox-rnwindows](https://raw.githubusercontent.com/react-native-community/react-native-checkbox/master/screenShots/demo-windows.png)
+
+The community module [react-native-checkbox](https://github.com/react-native-community/react-native-checkbox) has also received Windows support this release. CheckBox is a rather unique native control for us and we're happy to have it in.
+
+Special shoutout and thank you to the [community members who helped make this possible!](https://github.com/react-native-community/react-native-checkbox/pull/68)
+
+# The Hermes Engine
+
+<img width="300" height="300" style="float: none; margin: -30px 4px 0;" src="https://reactnative.dev/docs/assets/HermesLogo.svg">
+
+[The Hermes Engine](https://hermesengine.dev/) is an open-source JavaScript engine that's been built to optimize the performance of React Native apps. It's currently opt-in for Android and macOS and we're in the process of bringing it to React Native Windows as well.
+
+If you'd like to experiment with it in your Windows or macOS React Native apps, [head over to our guidance page](https://microsoft.github.io/react-native-windows/docs/hermes) to learn more about it.
+
+# What We've Been Doing
+
+Saying that working from home has taken some getting used to would be an understatement - it's been an incredible, eye-opening, stressful, and amazing adventure so far, obviously for many reasons both within and outside of work.
+
+However, that hasn't stopped us from trying to stay connected (virtually) with the passionate and awesome community that is React Native and cross-platform developers.
+
+Check out some of the events we've done and will be doing:
+
+## React Wednesdays with digitalprimates
+
+In mid August we connected with connected with the folks over at [CodeItLive](https://www.twitch.tv/codeitlive) to talk about React Native on Windows and macOS:
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=1wk-Id0fgqE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## React EU
+
+Looking a bit forward, check out the React Native EU conference hosted by Callstack, where our very own Steven Moyes will be presenting about React Native for Desktops and Dual Screens.

```

Comment:
Suggest linking to Steven's twitter handle?

Reply:
Steven added his handle!